stanford_profile.profile

```diff
@@ -255,3 +262,21 @@ function stanford_profile_xmlsitemap_link_alter(array &$link, array $context) {
 
   }
 }
+
+/**
```

Comment:
@sherakama ishi wanted to get rid of the core linkfieldwidget description. what do you think of this approach?

Reply:
to me this approach seems fine if it works. kinda overkill for such a little gain. this field is normally set only 1 time and rarely touched again

Reply:
Apparently there was a feeling that this was worth removing

Reply:
Works for me. I've been asked to remove this description from other fields throughout the accelerants as well.